### DISCUSSION TITLE
As in prisma query

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6253] [Q&A]
how can i give custom name to field in query
for example :
```sql
SELECT username as name from USER
```
how can i implement this in prisma?

**Comment by ruheni** [comment]
Hi @BishoyNarouz  👋🏾

This is possible using the `queryRaw` API. Your query will look something similar to this:

```ts
const result = await prisma.$queryRaw`SELECT username as name from USER`
```

Our docs provide more information on how to work with the [`queryRaw` API](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access/#queryraw). If you hit a snag, feel free to reach out. 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
